src/abstractdataframe/io.jl

```diff
@@ -156,6 +156,17 @@ function Base.show(io::IO, mime::MIME"text/html", dfr::DataFrameRow; summary::Bo
     _show(io, mime, view(parent(dfr), [r], c), summary=summary, rowid=r)
 end
 
+function Base.show(io::IO, mime::MIME"text/html", dfrs::DataFrameRows; summary::Bool=true)
+    write(io, "<p>DataFrameRows</p>")
+    _show(io, mime, parent(dfrs), summary=summary, rowid=r)
+end
+
+function Base.show(io::IO, mime::MIME"text/html", dfcs::DataFrameColumns{T,V};
+                   summary::Bool=true) where {T,V}
+    write(io, "<p>DataFrameColumns (with names = $(V <: Pair))</p>")
```

Comment:
Isn't this going to explode when there are many columns? Maybe just print `m×n DataFrameRows`, just like we do for `DataFrame`? Also, pass `summary=false`?

Reply:
Scratch that, I hadn't realized this is just a Boolean. Though my point still holds about printing the dimensions and passing `summary=false`.

Reply:
Agreed. There are many more things to fix but I just wanted to make sure that we agree on the general approach.